src/mam/mod_mam_rdbms_async_pool_writer.erl

```diff
@@ -28,9 +28,7 @@
 -behaviour(gen_mod).
 -behaviour(mongoose_module_metrics).
 
--export([archive_size/4,
-         archive_message/3,
-         lookup_messages/3]).
```

Comment:
The two removed callbacks were completely unused, right?

Reply:
right, they were used long time ago to wait for the writer, but we've removed that feature earlier.
there are no hook registration for it.